code/datums/components/pet_commands/pet_commands_basic.dm

```diff
@@ -141,6 +141,36 @@
 	controller.queue_behavior(attack_behaviour, BB_CURRENT_PET_TARGET, targetting_datum_key)
 	return SUBTREE_RETURN_FINISH_PLANNING
 
+/**
+ * # Breed command. breed with a partner!
+ */
+/datum/pet_command/point_targetting/breed
+	command_name = "Breed"
+	command_desc = "Command your pet to attempt to breed with a partner."
+	radial_icon = 'icons/mob/simple/animal.dmi'
+	radial_icon_state = "heart"
+	speech_commands = list("breed", "consumate")
```

Comment:
```suggestion
	speech_commands = list("breed", "consummate")
```
The mental image of pointing at your dog-sized bug and ordering it to consummate is an interesting one